database/config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     password: process.env.DATABASE_PASSWORD,
     database: process.env.TEST_DATABASE_NAME,
     host: process.env.DATABASE_HOST,
-    dialect: process.env.DATABASE_DIALECT
+    dialect: 'postgres'
```

Comment:
why the change here?

Reply:
Travis was complaining that it has to be hard-coded. I have included it in the .yml file and changed this back to process.env.DATABASE_DIALECT